clients/spark/build.sbt

```diff
@@ -42,7 +42,7 @@ buildInfoPackage := "io.treeverse.clients"
 enablePlugins(S3Plugin, BuildInfoPlugin)
 
 libraryDependencies ++= Seq(
-  "io.lakefs" % "sdk" % "1.0.0",
+  "io.lakefs" % "sdk" % "1.53.1",
```

Comment:
Let's have this as a separate PR!  It's not that I'm afraid, the lakeFS compatibility guarantees means this is a perfectly safe change.  But I'd still rather be careful because I'm actually scared by this change.

Reply:
Because we squash merges, 2 PRs give a more detailed commit history.  That makes it easier to `git bisect` and/or roll back some changes.  But not critical.